### DISCUSSION TITLE
Community Hyprland Matrix Space

### DISCUSSION THREAD

**Discussion by NovaAndrom3da** [hyprwm/Hyprland#1891] [General]
Join the matrix space over at https://matrix.to/#/#hyprland:naln1.ca (`#hyprland:naln1.ca`)

If your client doesn't support spaces:
`#hyprland-main:naln1.ca`
`#hyprland-offtopic:naln1.ca`
`#hyprland-q&a:naln1.ca`

**Comment by NovaAndrom3da** [comment]
@vaxerski I know you're busy, but maybe we could go about bridging these at some point? You don't have to worry about the matrix side, ofc.

**Reply by vaxerski** [reply]
no

**Reply by NovaAndrom3da** [reply]
Why?

**Reply by vaxerski** [reply]
because none of our current mod team want to moderate it, and it would require more work because a) no phone verif and b) no auto nsfw filter

**Reply by NovaAndrom3da** [reply]
if it's bridged to the discord, that also means that everything there is also present on the discord, meaning that any moderation actions on the discord also take into effect on the matrix rooms as well

what does phone verifiction have to do with anything though?

**Reply by vaxerski** [reply]
> meaning that any moderation actions on the discord also take into effect on the matrix rooms as well

No, we can't ban the user from discord, and discord's anti-nsfw filter doesn't work.

> what does phone verifiction have to do with anything though?

Discord: you wanna make 20 bot accounts? phone verification makes that way harder
Matrix: you wanna make 20 bot accounts? Feel free.

**Reply by NovaAndrom3da** [reply]
Fair enough, I suppose, but I would just like to say that I've never experienced any issue whatsoever with bots & spammers/trolls on matrix (like I have on discord)
